studio/src/__tests__/schema-helpers.test.ts

```diff
@@ -53,6 +65,21 @@ test("return the correct types with deprecated fields or args", async () => {
   expect(deprecated[1].fields?.[0]?.name).toEqual("fullName");
 });
 
+test("return the correct types with authenticated fields", async () => {
```

Comment:
I appreciate that the other tests do not adhere to the style, but `test` should read like English, e.g.,
`test('that something happens', ....`